docs/mnist_example/README.md

```diff
@@ -106,6 +106,53 @@ onnx-mlir -O3 --EmitJNI mnist.onnx
 
 A `mnist.jar` should appear, which corresponds to the compiled model object file along with Java API classes.
 
+## Multi-threading
+
+onnx-mlir provides a multi-thread safe parallel compilation mode. Whether each thread is given a name or not by the user, onnx-mlir is multi-threaded safe. If you would like to give a name to a thread, use the `-customEnvFlags` keyword and an example can be found as follows.
+
+```bash
+onnx-mlir -O3 -customEnvFlags mnistwith03 [--EmitLib] mnist.onnx -o mnist03
+```
+
+A multi-threaded experiment from command line written in Python is provided.
+
+```python
+import datetime
+import os
+import threading
+ 
+def execCmd(cmd):
+    try:
+        print("command " + cmd + " starts at " + str(datetime.datetime.now()))
+        os.system(cmd)
+        print("command " + cmd + " is finished at " + str(datetime.datetime.now()))
+    except:
+        print("command " + cmd + " meets errors")
+ 
+if __name__ == '__main__':
+    
+    # define 2 different commands
```

Comment:
Eventually, we should hide this into the PyRuntime interface, but let's do it in a subsequent PR